examples/primitives-images/index.html

```diff
@@ -14,14 +14,17 @@
         <img id="office3" src="mozvr-office-3.jpg">
       </a-assets>
 
-      <a-image position="0 1 -8" width="16" height="4.5" src="#office1"></a-image>
-      <a-image position="10 10 -20" width="14" height="9" scale="0.4 0.4 0.4" src="#office2"></a-image>
-      <a-image position="-6 6 -4" width="14" height="9" scale="0.4 0.4 0.4" src="#office2"></a-image>
-      <a-image position="-2 -5 -10" width="14" height="9" scale="0.4 0.4 0.4" src="#office2"></a-image>
-      <a-image position="-8 -1 -1" width="12" height="8" scale="0.5 0.5 0.5" src="#office3"></a-image>
-      <a-image position="5 -2 -3" width="12" height="8" scale="0.3 0.3 0.3" src="#office3"></a-image>
+      <a-image id="eventImage" position="0 1 -8" width="16" height="4.5" src="https://upload.wikimedia.org/wikipedia/commons/5/5e/Big_Wood,_N2.JPG">
+
+      </a-image>
 
       <a-sky color="#ECECEC"></a-sky>
     </a-scene>
+
+    <script type="text/javascript">
+      document.getElementById('eventImage').addEventListener('textureLoaded', function (e) {
```

Comment:
Is a message on console a good way to illustrate the feature? What about doing something visual when the textures load?